openhcl/underhill_attestation/src/secure_key_release.rs

```diff
@@ -116,6 +123,8 @@ pub async fn request_vmgs_encryption_keys(
     let mut timer = pal_async::timer::PolledTimer::new(&driver);
 
     for i in 0..max_retry {
+        tracing::info!(CVM_ALLOWED, "Attempt {} to get VMGS key-encryption key", i);
```

Comment:
Never use string interpolation in a `tracing` statement. Use key-value pairs. This makes it possible to group by target+message string when analyzing logs.

So something like `tracing::info!(CVM_ALLOWED, attempt = i, "attempt to get VMGS key-encryption key").